### DISCUSSION TITLE
Walkthrough install for MediaTemple (Grid) or Dreamhost

### DISCUSSION THREAD

**Discussion by paulsizer** [statamic/cms#5397] [Q&A]
Hi Everyone,

Does anyone have a walkthrough on how to install Statamic onto Dreamhost or MediaTemple Grid? I don't want to go down the WordPress route so thought I would give Statamic a try!

I have everything running locally fine.

Any help would be appreciated.

**Comment by edalzell** [comment]
What parts aren't working for you?

**Reply by paulsizer** [reply]
@edalzell i have copied the whole site folder across to media temple. 

But when I access it I just get 

Forbidden

You don't have permission to access this resource.

Do I need to install anything in particular?

**Reply by edalzell** [reply]
"site" folder? Shared hosting is hard, look up "laravel shared hosting"

**Reply by duncanmcclean** [reply]
One thing to check: are you pointing your site to the 'public' folder?

**Reply by paulsizer** [reply]
Thanks @edalzell and @duncanmcclean I'll take a look at both of those and see how I get on
